core/src/main/scala/pureconfig/ConfigConvert.scala

```diff
@@ -224,7 +226,8 @@ object ConfigConvert extends LowPriorityConfigConvertImplicits {
   }
 
   // traversable of types with an instance of ConfigConvert
-  implicit def deriveTraversable[T, F[T] <: TraversableOnce[T]](implicit configConvert: Lazy[ConfigConvert[T]],
+  implicit def deriveTraversable[T, F[T] <: TraversableOnce[T]](implicit
```

Comment:
Nitpick: Please manually move the second `implicit` down to the next line for consistency with other methods in the neighborhood.